### DISCUSSION TITLE
Black Screen while install Kali Linux on Macbook Air M1

### DISCUSSION THREAD

**Discussion by theanuraaag** [utmapp/UTM#4035] [General]
I am getting a black screen when I tried to install Kali Linux on my Macbook Air M1 using UTM, it works normally till the option install, graphical install, etc but when I click on Graphical Install a black screen appears and nothing happens after that.
<img width="1440" alt="Screenshot 2022-05-18 at 11 31 28 AM" src="https://user-images.githubusercontent.com/86941924/168968310-7d79bd34-59c1-4f56-9c0a-6e8cf80633b8.png">
<img width="1440" alt="Screenshot 2022-05-18 at 11 31 36 AM" src="https://user-images.githubusercontent.com/86941924/168968320-da2b90da-dfc2-4c38-bf39-a6fdfca4c5cb.png">
<img width="1440" alt="Screenshot 2022-05-18 at 11 32 51 AM" src="https://user-images.githubusercontent.com/86941924/168968345-11758afd-751f-4507-a388-f08e7b096255.png">

**Comment by Skywt2003** [comment]
Same issue here :(

**Comment by SpakeLee** [comment]
Same issue too. Orz

**Comment by shalor1k** [comment]
same issue for about 6 months now ;(

**Comment by ljis120301** [comment]
I'm also having the same issue

**Comment by Mvmo** [comment]
Same issue

**Comment by DylKhn** [comment]
I had the same issue on Kali 2022.2 but installing the previous version 2022.1 fixed the problem.

**Reply by theanuraaag** [reply]
@DylKhn do you able to install tor browser in it ?

**Reply by DylKhn** [reply]
This is where I got it from. 
[Link](http://old.kali.org/kali-images/kali-2022.1/)

Unsure if it's official or not.

**Reply by theanuraaag** [reply]
2022.1 arm64 ...right ?

**Reply by DylKhn** [reply]
Yep that's it.

**Reply by theanuraaag** [reply]
Thank you so much man for your time , I will try this and will let you know .

**Reply by Mvmo** [reply]
I've tried it and installed it. it's just working fine

**Reply by theanuraaag** [reply]
Nice

**Reply by theanuraaag** [reply]
@Mvmo Tor Browser requires a CPU with SSE2 support.
I am getting this error while installing Tor Browser in Kali Linux on Macbook Air M1 .

**Reply by DrewBrasher** [reply]
Thanks! I had the save issue and version 2022.1 arm64 worked for me also.

**Reply by riro707** [reply]
> @Mvmo Tor Browser requires a CPU with SSE2 support. I am getting this error while installing Tor Browser in Kali Linux on Macbook Air M1 .

Hi ! Have you found a solution for this ? I Want to install tor in my virtual machine (I have a macbook air m1 and my machine runs on Kali) and it tells me that my CPU doesn't support SSE2

**Reply by mandresy-ricco** [reply]
thanks it's works, I have a MacBook Pro M1 PRO

**Comment by theanuraaag** [comment]
@DylKhn can you please share previous version link 🙏

**Reply by Mvmo** [reply]
@theanuraaag https://cdimage.kali.org/kali-2022.1/kali-linux-2022.1-installer-arm64.iso

**Reply by theanuraaag** [reply]
Thanks

**Comment by theanuraaag** [comment]
@DylKhn can you please confirm that tor browser is working or not ?

**Comment by shalor1k** [comment]
@DylKhn can you please show your utm settings?

**Comment by Crane-Mocker** [comment]
Same issue when using kali-linux-2022.3-installer-arm64.iso

**Comment by mehdinourollah** [comment]
Just try Console Instead of Full in the display page of UTM settings for each VM and then try Install from console instead of GUI Install

**Reply by kdubovikov** [reply]
Can confirm that this worked for me

**Comment by mrthrt** [comment]
Same issue. downloaded 2022.3 arm64.iso. Issue happens as described above on both VMWare (latest version) and UTM (latest version). Downloaded 2022 arm64.iso and install is running perfectly on VMWare. Yet to try on UTM but i'm sure it will also install. Any fix would be appreciated, keen to be running 2022.3.

**Comment by Lit7leb0y** [comment]
Guys, you can just add a new display before starting the VM

**Comment by Lit7leb0y** [comment]
It should be on edit in the left corner

**Comment by Lit7leb0y** [comment]
Here's the guide: https://docs.getutm.app/guides/kali/

**Comment by wrobelda** [comment]
Looks like my issue is related: https://github.com/utmapp/UTM/discussions/4768

**Comment by elulcao** [comment]
Use the official instructions, and create a serial device.
https://docs.getutm.app/settings-qemu/devices/devices/

**Reply by mmercurio** [reply]
> Use the official instructions, and create a serial device.
> https://docs.getutm.app/settings-qemu/devices/devices/

This doesn't work for me on M1 MacBook Pro. The serial device works during install, but then on reboot the virtual VGA screen is always blank, even after removing the serial device.

**Comment by lucasballr** [comment]
UTM seems to have removed the ability to add a serial device to the VM. This is what the UI looks like now.
<img width="792" alt="Screenshot 2024-04-23 at 10 40 42 AM" src="https://github.com/utmapp/UTM/assets/63262961/db43e11c-8ba1-40fc-88d8-a692bef64505">

**Reply by DrewBrasher** [reply]
What version of UTM do you have? I had this issue a few days ago. I installed the latest version and the option to add a serial devices was back.
